File: backend/domain/dto/PetDataResponseDTO.java
package com.example._th_project.domain.dto;


import com.example._th_project.domain.table.Pets;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PetDataResponseDTO {

    private Long petId;
    private String petName;
    private String species;
    private String breed;
    private Integer age;
    private Double weight;
    private String gender;
    private byte[] petImg;

    public PetDataResponseDTO(Pets pet) {
        this.petId = pet.getPetId();
        this.petName = pet.getPetName();
        this.species = pet.getSpecies();
        this.breed = pet.getBreed();
        this.age = pet.getAge();
        this.weight = pet.getWeight();
        this.gender = pet.getGender();
        this.petImg = pet.getPetImg();
    }
}

File: frontend/android/model/PetDataResponseDTO.kt
package com.project.meongnyangcare.model

import com.google.gson.annotations.SerializedName

data class PetDataResponseDTO(
    @SerializedName("petId") val petId: Long,
    @SerializedName("petName") val petName: String,
    @SerializedName("species") val species: String,
    @SerializedName("breed") val breed: String,
    @SerializedName("age") val age: Int,
    @SerializedName("weight") val weight: Double,
    @SerializedName("gender") val gender: String,
    @SerializedName("petImg") val petImg: String // Base64 인코딩된 이미지 데이터
)
